docs/changelog/fragments/10.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add proxycommand support for session and
+update session execption to LibsshSessionException - by :user:`ganeshrn`
```

Comment:
JFYI the `:user:` role doesn't work yet, I'll need to add it to extlinks in conf.py, but ignore it for now. You can use this but sphinx build will fail (for now).